Project/js-practice/rock-paper-scissors/index.js

```diff
@@ -0,0 +1,115 @@
+let score = JSON.parse(localStorage.getItem('score')) || {
+    wins: 0,
+    losses: 0,
+    ties: 0
+  };
+  
+updateScoreElement();
+
+let isAutoPlaying = false
+let intervalID;
+function autoPlay() {
+    if (!isAutoPlaying){
+        intervalID = setInterval(() => {
```

Comment:
@codal-kcolaco What is this used for setInverval and setTimeout is used for?

Reply:
**setTimeout** is used to execute a piece of code once after a specified delay.
**setInterval** is used to repeatedly execute a piece of code at specified intervals.